apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -444,6 +444,53 @@ function buildSelectTopQueries(options: STQOptions): STQResults {
   }
 }
 
+function buildParameterizedSelectTopQuery(
```

Comment:
Hey, it's not a good idea to have duplicate code like this, if we need to fix a bug, we'd have to fix it twice. Feels like you're only going with this solution because it was quick.

It would be better to make `buildSelectTopQueries` take an additional parameter like `inlineParams = false`.

Seems like the only thing that needs to be changed is the calculation of `value` during the filter creation.

Then you can add some tests for `buildSelectTopQueries` with both sets of flags.

Reply:
Sorry, that was a careless decision of me, I don't know what I was doing 😞 .

Reply:
Not a big deal :-)